addons/project/wizard/project_share_wizard.py

```diff
@@ -43,6 +63,10 @@ def _compute_resource_ref(self):
             else:
                 wizard.resource_ref = None
 
+    @api.depends('subject')  # fake trigger otherwise not computed in new mode
+    def _compute_render_model(self):
+        self.render_model = 'project.share.wizard.template'
```

Comment:
if this line is not present, the render_body method will fail because the render_model is set to False, instead of the correct model. I do not understand why it is set to False, despite the fact that i've set the correct model in the template, but it looks like it is a know issue. This trick was already present somewhere else in the code base.